libs/service/api_service.cpp

```diff
@@ -431,6 +432,73 @@ ebpf_verify_and_load_program(
 
 #endif
 
+_Must_inspect_result_ ebpf_result_t
+ebpf_authorize_native_module(_In_ HANDLE native_image_handle) noexcept
```

Comment:
thunk/platform.h and thunk/mock/mock.h (and maybe other files) took the viewpoint that HANDLE is a value not a pointer per se, and so `_In_` isn't used with it.